src/Catty/PlayerEngine/Sensors/Heading/CompassDirectionSensor.swift

```diff
@@ -39,14 +39,21 @@
     }
 
     func rawValue(landscapeMode: Bool) -> Double {
-        self.getLocationManager()?.heading?.magneticHeading ?? type(of: self).defaultRawValue
+        if !landscapeMode {
+            return self.getLocationManager()?.heading?.magneticHeading ?? type(of: self).defaultRawValue
+        } else {
+            let rawValuePortrait = self.getLocationManager()?.heading?.magneticHeading ?? type(of: self).defaultRawValue
+            let rawValueLandscape = rawValuePortrait - 90.0
```

Comment:
In case this sensor is not available, the default value is wrong. Please verify and document this case in the tests.

Reply:
Thanks for your review, I changed it according to your request !